posthog/models/error_tracking/error_tracking.py

```diff
@@ -20,6 +23,37 @@ class Status(models.TextChoices):
     name = models.TextField(null=True, blank=True)
     description = models.TextField(null=True, blank=True)
 
+    def merge(self, issue_ids: list[str]) -> None:
+        override_inserts: list[tuple[str, int, str]] = []
+
+        with transaction.atomic():
+            for issue_id in issue_ids:
+                fingerprints = resolve_fingerprints_for_issue(team_id=self.team.pk, issue_id=issue_id)
+
+                for fingerprint in fingerprints:
+                    override_inserts.append(
+                        update_error_tracking_issue_fingerprint(
+                            team_id=self.team.pk, issue_id=self.id, fingerprint=fingerprint
+                        )
+                    )
+
+            ErrorTrackingIssue.objects.filter(team=self.team, id__in=issue_ids).delete()
```

Comment:
Do we ever want to delete an issue? Feels like asking for user pain. Auto-archiving and updating the description to say "Issue merged into $self - original description:\n" might be nicer?

We could go a step further, and let this function return the OLD fingerprint->issue_id mapping, so that we could add an "oops" button to the UI that lets users reverse a merge they just did (and we'd definitely need to keep the old issue around then) - I think we should actually build that, but not in this PR.

Reply:
Lets stick with this, and if we get support about losing issue descriptions or history (a thing I've thought about is we might want an ability to track the assignment or status changes over time associated with an issue) we can revisit. I expect we'll have to switch to a more elaborate approach eventually but for now let's just :shipit:.